tailwind.config.js

```diff
@@ -194,6 +200,9 @@ export default {
       lineHeight: {
         17: "17px",
       },
+      boxShadow: {
+        '2px-white': '0 0 0 2px #ffffff',
```

Comment:
where is this being used ?

Reply:
Initial focus of the dropdown in the OPM sidebar upon opening

Reply:
Can you share the filename and line where this is being used ?

Reply:
index.css line 296
#custom-hover.initial-focus .p-dropdown.p-focus {
  @apply !shadow-2px-white;
}